components/input/InputBase.js

```diff
@@ -0,0 +1,191 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import omit from 'lodash.omit';
+import Box, { omitBoxProps, pickBoxProps } from '../box';
+import { ErrorText, HelpText } from '..';
+import theme from './theme.css';
+
+export default class InputBase extends PureComponent {
```

Comment:
When do you choose to 
```js
export default class X extends Y {}
``` 
and when to do it as
```js
class X extends Y {}
export default X;
```

Reply:
Didn't really think about it tbh, the second notation is what we normally use, so I'll follow that convention.